codeclimate-validate-config/bin/run

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+#
+# TODO: validate engine names as valid engines. this requires access to the
+# shared engine registry, so we should think carefully if that functionality is
+# worth the coupling.
```

Comment:
There's a bit of an edge case with this idea for when users are locally developing new engines. Maybe not worth thinking about that edge case. Even ignoring that I feel this is kind of a non-feature and not very worth the coupling & implementation effort.